File: works/tmp/1475.kt
import java.util.*


fun main()
{
    val sc = Scanner(System.`in`)
    val number = sc.next()
    var nums = Array(10) {0}
    for(i in 0 until number.length) nums[number[i]-'0']++
    var a = 0
    var b = 0
    for(i in 0 until 10)
    {
        if(i == 6 || i == 9) continue
        a = Math.max(a, nums[i])
    }

    b = nums[6]+nums[9]
    if(b % 2 != 0) b = b / 2 + 1
    else b /= 2
    print(Math.max(a,b))
}

File: works/tmp/1475.cs
using static System.Console;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.IO;
using System;

namespace Project1
{
    public class Class1
    {

        public static void Main(string[] args)
        {
            StreamReader sr = new StreamReader(Console.OpenStandardInput());
            string number = sr.ReadLine();
            int[] nums = new int[10];
            for (int i = 0; i < number.Length; i++) nums[number[i] - '0']++;
            int a = 0, b = 0;
            for(int i = 0; i < 10; i++)
            {
                if (i == 6 || i == 9) continue;
                a = Math.Max(a, nums[i]);
            }
            b = nums[6] + nums[9];
            if (b % 2 != 0) b = b / 2 + 1;
            else b /= 2;
            Write(Math.Max(a, b));
        }
    }
}
